specification/changeanalysis/resource-manager/Microsoft.ChangeAnalysis/preview/2020-04-01-preview/changeanalysis.json

```diff
@@ -0,0 +1,432 @@
+          },
+          "204": {
+            "description": "No Content - the specified resource was not found."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ChangeAnalysis/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "summary": "Lists operations for the resource provider.",
+        "description": "Lists all the supported operations by the Microsoft.ChangeAnalysis resource provider along with their description.",
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json",
+          "default"
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderOperationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ResourceIdentity": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "None",
+            "SystemAssigned"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ManagedIdentityTypes",
+            "modelAsString": true
+          }
+        },
+        "principalId": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true
+        },
+        "tenantId": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AzureMonitorWorkspaceProperties": {
+      "type": "object",
+      "properties": {
+        "workspaceId": {
+          "type": "string"
+        },
+        "workspaceResourceId": {
+          "type": "string"
+        }
+      }
+    },
+    "NotificationsState": {
+      "enum": [
+        "None",
+        "Enabled",
+        "Disabled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "NotificationsState",
+        "modelAsString": true
+      }
+    },
+    "NotificationSettings": {
+      "type": "object",
+      "properties": {
+        "azureMonitorWorkspaceProperties": {
+          "$ref": "#/definitions/AzureMonitorWorkspaceProperties"
+        },
+        "activationState": {
+          "$ref": "#/definitions/NotificationsState"
+        }
+      }
+    },
+    "ConfigurationProfileResourceProperties": {
+      "type": "object",
+      "properties": {
+        "notifications": {
+          "$ref": "#/definitions/NotificationSettings"
+        },
+        "lastModifiedBy": {
```

Comment:
Please see: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

These metadata properties are being rolled out for all resources in a top-level "systemData" property. This will be mandatory at some time in the future. Please implement the created/modified metadata properties consistently with this new standard

Reply:
Thanks I talked with Jenny Hunter about this and Gaurav, will update shortly -

Reply:
@KrisBash , all the changes applied and required checks passed , please review and sign off.